src/circuit.ts

```diff
@@ -226,13 +229,13 @@ export class Circuit extends EventEmitter {
     litNetwork?: LIT_NETWORKS_KEYS,
     pkpContractAddress = PKP_CONTRACT_ADDRESS,
     pkpHelperContractAddress = PKP_HELPER_CONTRACT_ADDRESS,
-    errorHandlingModeStrict: boolean = false,
+    errorHandlingModeStrict = false,
   ) {
     super();
     this.errorHandlingModeStrict = errorHandlingModeStrict;
     this.signer = signer ? signer : ethers.Wallet.createRandom();
     this.litClient = new LitJsSdk.LitNodeClient({
-      litNetwork: litNetwork ?? 'cayenne',
+      litNetwork: litNetwork ?? "cayenne",
```

Comment:
Maybe we can default to datil dev as cayenne is not supported anymore in latest sdks